WebKitBrowser/CMakeLists.txt

```diff
@@ -15,39 +15,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-cmake_minimum_required(VERSION 3.3)
-find_package(WPEFramework)
-
-list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/../cmake/")
-
-include(CmakeHelperFunctions)
-
-# Library installation section
-string(TOLOWER ${NAMESPACE} STORAGE_DIRECTORY)
-
-if(BUILD_REFERENCE)
-    add_definitions(-DBUILD_REFERENCE=${BUILD_REFERENCE})
-endif()
+project(WebKitBrowser)
 
 set(PLUGIN_NAME WebKitBrowser)
 set(MODULE_NAME ${NAMESPACE}${PLUGIN_NAME})
 
-find_package(ProxyStubGenerator REQUIRED)
-find_package(JsonGenerator REQUIRED)
-
-ProxyStubGenerator(INPUT ${CMAKE_CURRENT_SOURCE_DIR}/interfaces)
-JsonGenerator(CODE STUBS
-  INPUT ${CMAKE_CURRENT_SOURCE_DIR}/WebKitBrowserPlugin.json
-  IFDIR  ${CMAKE_CURRENT_SOURCE_DIR}/interfaces/json/
-  OUTPUT ${CMAKE_CURRENT_SOURCE_DIR}/interfaces/json/
-)
-
-# JsonGenerator(DOCS
```

Comment:
My 2 cents, lets then start moving the interfaces into a repository, the interfaces are already drifting apart since they reside in two places. Let me pick this up with Wouter/RDK that we get a repository. As this is the master and you are now on the sprint, I guess we must be able to resolve this before you guys go move to the next sprint and this put some pressure on getting the interfaces out of Thunder :-)

Reply:
Sounds good. As for the sprint, this change will need to be in sync with the Thunder update as well.